docs/root/manual/getting-started/run-signer-node.md

```diff
@@ -454,10 +581,84 @@ sudo iptables -L -v
 sudo service netfilter-persistent save
 ```
 
+:::tip
+
+Here is an example command created with the example set for `release-preprod`:
+
+```bash
+sudo iptables -A INPUT -s 192.168.1.75 -p tcp --dport 3128 -j ACCEPT
+sudo iptables -L -v
+sudo service netfilter-persistent save
+```
+
+:::
+
 ## Verify the Mithril Signer deployment
 
 :::tip
 There is a delay of `2` epochs between the registration of the signer node and its ability to generate individual signatures. This delay is further explained in the [Mithril certificate chain in depth](https://mithril.network/doc/mithril/mithril-protocol/certificates) documentation.
 
 Once this delay has passed, you should be able to observe your `PoolId` listed in some of the certificates accessible on the [`Mithril Explorer`](https://mithril.network/explorer).
 :::
+
+### Verify your signer is registered
+
+After installing the Mithril Signer, you can verify that your node is registered by checking your Mithril Signer node logs.  
+
+First, download the script in the desired directory:
+
+```bash
+wget "https://raw.githubusercontent.com/input-output-hk/mithril/main/docs/scripts/verify_signer_registration.sh"
```

Comment:
- The URL should be on the doc website `mithril.network`
- Also no need to add double quotes around the URL

Reply:
For the first point, I'm not sure to get it. The URLs should be : 
https://mithril.network/doc/scripts/verify_signer_registration.sh and https://mithril.network/doc/scripts/verify_signer_signature.sh ?

Reply:
I think the scripts should probably be put in the the `static` folder :thinking: 

Could you do the following:
- move them in `static/scripts` folder
- verify that you can download locally the scripts at `http://localhost:3000/doc/scripts/verify_signer_registration.sh` and `http://localhost:3000/doc/scripts/verify_signer_signature.sh`
- replace the url with `https://mithril.network/doc/scripts/verify_signer_registration.sh` and `https://mithril.network/doc/scripts/verify_signer_signature.sh`